vm/src/exceptions.rs

```diff
@@ -1621,3 +1622,9 @@ pub(super) mod types {
         "Base class for warnings about encodings."
     }
 }
+
+impl ToPyException for ReprOverflowError {
+    fn to_pyexception(&self, vm: &VirtualMachine) -> PyBaseExceptionRef {
+        vm.new_overflow_error(String::from("This object has overflow repr"))
```

Comment:
`PyStr` do this like `vm.new_overflow_error(err.to_string())`. Using same code will be better.

Reply:
I just fixed the new code to follow this convention. However, this leaves `impl ToPyException for ReprOverflowError` unused, but I won't remove this in case of further usage.

Reply:
We can keep your original version in bytesinner and change this to_pyexception inside to reuse err.to_string().